code/style.css

```diff
@@ -1,4 +1,188 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+a {
+  font-family: "helvetica neue";
+  text-decoration: none;
+  color: inherit;
 }
+
+a:hover {
+  border-bottom: 3px dotted #ff7a59;
+}
+
+/* styling for header and navbar */
+
+header {
+  background: white;
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 10px;
+}
+
+header img {
+  height: 50px;
+  width: 50px;
+}
+
+nav {
+  padding:15px;
+ }
+ 
+ article {
+  background:white;
+  height: 800px;
+}
+
+ /* big news article styling */
+
+.bignews-text h2 {
+  color: black;
+  text-align: left;
+    padding-left: 25px;
+    width: 100%;
+}
+
+.big-news {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.bignews-img {
+  display: flex;
+  width: 50%;
+  border: black;
+}
+
+.bignews-img img {
+  width: 500px;
+  max-width: 100%;
+  max-height: 100%;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: auto;
+  margin-bottom: auto;
```

Comment:
Since all of the properties are set to `auto`, we can just type one declaration `margin: auto`